dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/listener/AggregateDataSetSMSListener.java

```diff
@@ -111,10 +111,10 @@ protected SMSResponse postProcess( IncomingSms sms, SMSSubmission submission )
         String per = subm.getPeriod();
         UID aocid = subm.getAttributeOptionCombo();
 
-        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.uid );
-        User user = userService.getUser( subm.getUserID().uid );
+        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.getUID() );
+        User user = userService.getUser( subm.getUserID().getUID() );
```

Comment:
Change to `.getUid()` consistently.

Reply:
@larshelge - as the change turned out to be quite big I've separated it into another PR here: https://github.com/dhis2/dhis2-core/pull/5421